src/client/app/shared/safe-image.component.ts

```diff
@@ -25,7 +27,7 @@ export class SafeImageComponent {
 				this.defaultPicSrc = config.defaultImage;
 			});
 
-		if (_.isString(this.imgSource) && this.imgSource.trim().length > 0) {
+		if (StringUtils.isInvalid(this.imgSource)) {
```

Comment:
Nice. This will make it easier if we ever want to migrate away from lodash too. Or if we want to switch to importing lodash operators by function (which would be a good thing to do eventually)